source/Module/Common/Common.psm1

```diff
@@ -13,6 +13,17 @@ enum severity
     high
 }
 
+<#
+    The category enum is used to define category and will assist in converting to [severity]
+    Do not change the order of either the severity enum or the category enum
+#>
+enum category
```

Comment:
If we define category as an enum can we use that enum to define the parameter type instead of using [string] and a validateSet?

Reply:
I like this idea, I'll start implementing it now.